config/presets/fastnet.go

```diff
@@ -54,11 +54,11 @@ func fastnet() config.Config {
 	conf.POST.BitsPerLabel = 8
 	conf.POST.K1 = 12
 	conf.POST.K2 = 4
-	conf.POST.LabelsPerUnit = 32 // bytes
+	conf.POST.LabelsPerUnit = 128 // bytes
 	conf.POST.MaxNumUnits = 4
 	conf.POST.MinNumUnits = 2
 	conf.POST.N = 32
-	conf.POST.B = 2
+	conf.POST.B = 16
```

Comment:
Since `B` will now always be 16, should we get rid of it as a parameter?

Reply:
The post-proving and verification code technically still supports B other than 16 so I'm leaving it for now. I will remove it once we finish work on the post and stabilize the algo.